pandasai/llm/cohere.py

```diff
@@ -0,0 +1,115 @@
+"""Cohere LLM"""
+
+import os
+from dotenv import load_dotenv
+import cohere
+from .base import LLM
+from ..exceptions import APIKeyNotFoundError, UnsupportedCohereModelError
+
+load_dotenv()
+
+
+class CohereLLM(LLM):
+    """Cohere LLM"""
+
+
+    _supported_chat_models = [
+        'command',
+        'command-nightly',
+        'command-light',
+        'command-light-nightly',
+        'command-xlarge',
+        'command-xlarge-nightly'
+    ]
+
+    api_token: str
+    model: str = "command-xlarge-nightly"
+    temperature: float = 0
+    max_tokens: int = 512
+    top_p: float = 1
+    frequency_penalty: float = 0
+    presence_penalty: float = 0.6
+    stop: str = None
+
+
+    def __init__(
+        self,
+        api_token: str = None,
+        **kwargs,
+    ):
+        self.api_token = api_token or os.getenv("COHERE_API_KEY")
```

Comment:
put the name of the environment variable in the [example file](https://github.com/gventuri/pandas-ai/blob/main/.env.example)